cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mysql/MySQLBackendImpl.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+ * Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
```

Comment:
Regarding copyright date in headers, two typical patterns are used:

* Set date at file creation, then keep that date forever (the one used at Orion repository, by the way)
* Add new dates each times you edit the file in consequitive order, e.g:

"Copyright 2015, 2016 Telefonica Investigación y Desarrollo, S.A.U"

or

"Copyright 2015-2016 Telefonica Investigación y Desarrollo, S.A.U"

Just changing a year for a new other is a kind of "lying" as that file has copyrigthed parts in previous years.

Not very important, just mentioning if you want to have a though on it...